userguide/tutorials/overdue.adoc

```diff
@@ -512,6 +512,139 @@ Even if the payment retry schedule is not configured explicitly, there is a defa
+                <autoReevaluationInterval>
+                 <unit>DAYS</unit>
+                 <number>4</number>
+                </autoReevaluationInterval>
+|isClearState   |Tag within `state`   |Specifies whether the current state is a clear state. A true/false value needs to be specified. |
+[source, xml]
+ <isClearState>true</isClearState>
+|enterStateEmailNotification   |Tag within `state`   |Deprecated|
+NA
+|===
+
+=== Overdue Conditions
+
+The following table lists the various overdue conditions. Note that it is also possible to combine multiple conditions in which case _ALL_ the specified conditions need to evaluate to true for the account to enter the particular overdue state.
+
+[cols="a,a,a"]
+|===
+|*Condition*   |*Description* |*Example*
+//-------------
+|numberOfUnpaidInvoicesEqualsOrExceeds   |Specifies that the overdue flow should kick in when the number of unpaid invoices equals or exceeds the specified value. An integer value needs to be specified.|  
+[source, xml]
+<condition>
+<numberOfUnpaidInvoicesEqualsOrExceeds>2
+</numberOfUnpaidInvoicesEqualsOrExceeds>
+</condition>
+|totalUnpaidInvoiceBalanceEqualsOrExceeds   | Specifies that the overdue flow should kick in when the total unpaid invoice balance equals or exceeds the specified value. A decimal value needs to be specified.  |
+[source, xml]
+<condition> 
+  <totalUnpaidInvoiceBalanceEqualsOrExceeds>1.00
+   </totalUnpaidInvoiceBalanceEqualsOrExceeds>
+</condition> 
+|timeSinceEarliestUnpaidInvoiceEqualsOrExceeds   |Specifies that the overdue flow should kick in when the specified duration has elapsed since the last unpaid invoice. The duration needs to be specified with a `unit` (one of `DAYS`, `WEEKS`, `MONTHS`, `YEARS`, `UNLIMITED`) and an integer value  | 
+[source, xml]
+<condition>
+  <timeSinceEarliestUnpaidInvoiceEqualsOrExceeds>
+    <unit>DAYS</unit>
+    <number>10</number>
+  </timeSinceEarliestUnpaidInvoiceEqualsOrExceeds>
+</condition>
+|responseForLastFailedPaymentIn|Specifies that the overdue flow should kick in when payment fails and the failed response is one of the specified values.  Possible values are defined in https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PaymentResponse.java[PaymentResponse]. Multiple values may be specified.    |   
+[source, xml]
+<condition>
+  <responseForLastFailedPaymentIn>
+    <response>INVALID_CARD</response>
+    <response>LOST_OR_STOLEN_CARD</response>
+     <response>INSUFFICIENT_FUNDS</response>
+  </responseForLastFailedPaymentIn>
+</condition>
+|controlTagInclusion   |Specifies that the overdue flow should kick in when the account is tagged with the specified control tag. Note that this condition cannot be specified by itself and can only be specified along with one of the other conditions as shown. | 
+[source, xml]
```

Comment:
Again, could you confirm if this is correct? I found that when the `controlTagInclusion` condition is specified by itself, the overdue flow is not triggered. It only works when the  `controlTagInclusion` condition is used along with some other condition.

Reply:
Weird: I would expect it to behave just like any other condition. https://github.com/killbill/killbill/blob/00d9ace820bc7ddf6961a16a2727abe17b198bf9/overdue/src/main/java/org/killbill/billing/overdue/config/DefaultOverdueCondition.java#L69-L84

Not a blocker for this PR but might be worth debugging why it's happening.

Reply:
I've updated the document for now, will investigate this sometime later.